scipy/stats/_continuous_distns.py

```diff
@@ -5748,19 +5748,27 @@ class ncf_gen(rv_continuous):
                                (-\lambda v_1 \frac{x}{2(v_1 x+v_2)})}
                               {B(v_1/2, v_2/2)  \gamma(\frac{v_1+v_2}{2})}
 
-    for :math:`n_1 > 1`, :math:`n_2, \lambda > 0`.  Here :math:`n_1` is the
+    for :math:`n_1, n_2 > 0`, :math:`\lambda\geq0`.  Here :math:`n_1` is the
```

Comment:
Take a look at "compiled" version of the [docs](https://docs.scipy.org/doc/scipy/reference/tutorial/stats/continuous_ncf.html) about non-central Fisher distribution. In the first line of the text: there are both used \nu_1, \nu_2 and v_1 and v_2 in the same context. This is typo, I think. I just suggested to use \nu_i instead of wrong v_1 and v_2 within this document only.

Reply:
ah I see now, they are 2 different docs, 1 is the tutorial, where the type was, that explains, thanks.